src/drop.test-d.ts

```diff
@@ -0,0 +1,48 @@
+import { drop } from "./drop";
+import { pipe } from "./pipe";
+
+describe("data first", () => {
```

Comment:
**suggestion:** can we add a test for `drop([1, 2, 3, 4, 5], 2 as number)`?

Reply:
added